near/omni-bridge/src/storage.rs

```diff
@@ -160,14 +160,26 @@ impl Contract {
         );
         self.accounts_balances.insert(&account_id, &storage);
 
-        if let Some(promise_id) = &self.init_transfer_promises.get(&account_id) {
-            let result = env::promise_yield_resume(promise_id, &[]);
-            env::log_str(&format!("Init transfer resume. Result: {result}"));
-        }
+        self.resume_promise(&account_id);
 
         storage
     }
 
+    #[private]
+    pub fn resume_promise(&self, account_id: &AccountId) -> PromiseOrValue<()> {
+        if let Some(promise_id) = &self.init_transfer_promises.get(account_id) {
+            let result = env::promise_yield_resume(promise_id, &[]);
+            env::log_str(&format!("Resume promise. Result: {result}"));
+
+            if !result {
+                return Self::ext(env::current_account_id())
+                    .resume_promise(&account_id)
```

Comment:
verify if it is ok to create the promise without fixed gas

Reply:
https://testnet.nearblocks.io/txns/5RCmMaBkWHnXcYtQJzrgdqZ36R8q6orGFKNHdZG5MJyt?tab=summary